.gitignore

```diff
@@ -31,19 +31,14 @@ storybook-static
 
 # misc
 .DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
 .eslintcache
 
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
 # IDEs
-.idea/*
-!.idea/runConfigurations
```

Comment:
This was to support having runConfigurations checked in. Was there any specific issues?

Reply:
removed for now as undesirable to maintain